### PR TITLE
Express boiler plate to kick start the VendorDirectory

### PR DESCRIPTION
**Pull request by firebase007**
Express boiler plate to kick start the VendorDirectory

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -1,9 +1,83 @@
...
+  res.render('register');
 });
 
+//login
+router.get('/login', function (req, res) {
+  res.render('login');
+});
+
+router.post('/register',function (req, res) {
+  //console.log(req.body)
```
Comment (lumiscript) [inline routes/users.js:22]: Remove console.logs if it doesnt describe anything.

File: models/users.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+module.exports.createUser = function (newUser, callback) {
+    bcrypt.genSalt(10, function (err, salt) {
+        bcrypt.hash(newUser.password, salt, function (err, hash) {
+            // Store hash in your password DB. 
+            newUser.password = hash;
+            newUser.save(callback);
+        });
+    });
+}
+//custom functions for accessing the database
```
Comment (lumiscript) [inline models/users.js:42]: consider creating a `helpers` directory and all these functions can live there.

Comment (firebase007) [reply models/users.js:42]: Updated to reflect a new helpers dir

File: routes/users.js (hunk 1 above)
Comment (firebase007) [reply routes/users.js:22]: Done

**Pull request merged**